editor/scene_tree_dock.cpp

```diff
@@ -1373,77 +1373,83 @@ void SceneTreeDock::_create() {
 		}
 
 	} else if (current_option == TOOL_REPLACE) {
-		Node *n = scene_tree->get_selected();
-		ERR_FAIL_COND(!n);
+		List<Node *> selection = editor_selection->get_selected_node_list();
+		if (selection.size() <= 0) {
```

Comment:
Changes look fine, but instead of this `if` + `return` it should be `ERR_FAIL_COND(selection.size() <= 0);` (i.e. if there was no selection when this option was triggered, it's a bug).

Reply:
So what you mean is I should change ```if + return```  to ```ERR_FAIL_COND(selection.size() <= 0)```?

Reply:
I have already fixed.